src/axom/sidre/tests/sidre_group.cpp

```diff
@@ -571,6 +574,55 @@ TEST(sidre_group, child_lists)
   delete ds;
 }
 
+//------------------------------------------------------------------------------
+// Test Group's list format for holding contents of a vector of strings.
+//------------------------------------------------------------------------------
+TEST(sidre_group, string_list)
+{
+  // Round-trip test from std::vector<std::string> to Group and back.
+  DataStore* ds = new DataStore();
+  Group* root = ds->getRoot();
+
+  std::vector<std::string> str_vec;
+  str_vec.push_back("This");
+  str_vec.push_back("is");
+  str_vec.push_back("a");
+  str_vec.push_back("vector");
+  str_vec.push_back("to");
+  str_vec.push_back("test");
+  str_vec.push_back("strings");
+  str_vec.push_back("in");
+  str_vec.push_back("sidre::Group's");
+  str_vec.push_back("list");
+  str_vec.push_back("format");
+
+  // my_strings is a Group in list format.
+  Group* my_strings = root->createGroup("my_strings", true);
+
+  // Put strings into the Group.
+  for(auto itr = str_vec.begin(); itr != str_vec.end(); ++itr)
+  {
+    View* str_view = my_strings->createViewString("", *itr);
```

Comment:
Does the `path` parameter need to be an empty string for this to work?

If so, can you please add a comment to that effect?
Here, and maybe as a doxycomment on the `createViewString` function?

See: https://github.com/LLNL/axom/blob/cab5877121dbc467a716237eef1128c861a4baba/src/axom/sidre/core/Group.cpp#L170-L189

Reply:
It does not need to be empty, but a non-empty string will have no effect and is not useable for accessing the `View` later on.  This is documented somewhere, but I'll look at making it more clear.

Reply:
Thanks -- in that case, it might be sufficient to just have a comment here to that effect, e.g.:
```cpp
// The first parameter will be ignored for list collections, so we use the empty string
```

Reply:
I'll add that comment and also create an issue to cover this more thoroughly in documentation.